applications/jupyter/workloads.tfvars

```diff
@@ -15,26 +15,32 @@
 ##common variables  
 ## Need to pull this variables from tf output from previous platform stage
 project_id = "ai-on-gke-jss-sandbox"
+
 ## this is required for terraform to connect to GKE master and deploy workloads
+cluster_name     = "ml-cluster"
+cluster_location = "us-central1"
+cluster_membership_id = "" # require for private cluster, default: cluster_name
 
 #######################################################
 ####    APPLICATIONS
 #######################################################
 
 ## JupyterHub variables
-namespace = "rag"
+namespace = "ml"
 gcs_bucket = "gcs-bucket"
 k8s_service_account = "default"
+gcp_service_account ="jupyter-service-account1"
```

Comment:
can we remove the "1" at the end?
can we make the names consistent between ray/jupyter? like call both vars gcp_service_account and set the values to "ray-system-account" and "jupyter-system-account"?